run-custom-mao.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -xe
+
+# This script needs an machine-api-operator image as an input parameter
+# For example: ./run-custom-mao.sh quay.io/sdasu/machine-api-operator:sdasu-fix
+export SCRIPTS_DIR=$(pwd)
+export CUSTOM_MAO_BUILD=$1
+export KUBECONFIG=ocp/auth/kubeconfig
+REPLACEMENT_TEXT="image: ${CUSTOM_MAO_BUILD}"
+MAO_REPO="raw.githubusercontent.com/sadasu/machine-api-operator"
+MAO_BRANCH="sdasu-fix"
+MAO_RBAC_MANIFEST="install/0000_30_machine-api-operator_09_rbac.yaml"
+
+oc --config ${KUBECONFIG} get secrets --namespace openshift-machine-api | grep mariadb
+
+# Stop CVO, MAO, machine-api-controller and Metal3 deployments
+oc --config ${KUBECONFIG} scale deployment --replicas=0 cluster-version-operator --namespace openshift-cluster-version
+oc --config ${KUBECONFIG} scale deployment --replicas=0 machine-api-operator --namespace openshift-machine-api
+
+wget https://${MAO_REPO}/${MAO_BRANCH}/${MAO_RBAC_MANIFEST} -O machine-api-operator-roles.yaml
+
+# Give the machine-api-operator rbac access to create and read secrets
+oc --config ${KUBECONFIG} apply -f machine-api-operator-roles.yaml
+
+oc --config ${KUBECONFIG} get deployments -n openshift-machine-api machine-api-operator -o yaml > mao.yaml
```

Comment:
we should put the downloaded mao.yaml into the ocp dir?

Reply:
Done.